app/src/main/java/ai/elimu/vitabu/BaseApplication.kt

```diff
@@ -1,40 +1,14 @@
 package ai.elimu.vitabu
 
 import android.app.Application
-import android.speech.tts.TextToSpeech
-import android.speech.tts.TextToSpeech.OnInitListener
 import android.util.Log
+import dagger.hilt.android.HiltAndroidApp
 
+@HiltAndroidApp
 class BaseApplication : Application() {
 
-    private val TAG = javaClass.name
-
-    var tts: TextToSpeech? = null
```

Comment:
@tuancoltech With the `TextToSpeech` instance gone, how will developers be able to do things like calling `tts.setSpeechRate()` and `tts.setLanguage()`? Will they still have access to the TTS instance somehow?

Reply:
@jo-elimu For the current version, those APIs are not available.
However, we can also add supports for the APIs and release a new version, to expose them to developers.
I'll add that in upcoming release.